src/Sylius/Behat/spec/Context/Transform/LexicalContextSpec.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class LexicalContextSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith();
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Behat\Context\Transform\LexicalContext');
+    }
+
+    function it_implements_context_interface()
+    {
+        $this->shouldImplement(Context::class);
+    }
+
+    function it_transforms_price_string_to_integer()
+    {
+        $this->getPriceFromString('10.00')->shouldReturn(1000);
+        $this->getPriceFromString('0.30')->shouldReturn(30);
+        $this->getPriceFromString('0.0134')->shouldReturn(1);
```

Comment:
If the system does not support such precision it should inform the developer by throwing exception and do not round it without asking. There is no point in writing "0.01234" in your scenario.